webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2959,6 +2970,38 @@ var userOverride = {
             _this.phoneNumberValidation = function (e) {
                 e.target.value = e.target.value.match(/^([0-9]{0,15})/)[0];
             };
+
+            _this.createSelectFieldDropdown = function (
+                options,
+                selectElement
+            ) {
+                var dropDownOption = document.createElement('option');
+                dropDownOption.setAttribute('value', '');
+                dropDownOption.textContent =
+                    MCK_LABELS['lead.collection'].option +
+                    ' ' +
+                    selectElement
+                        .getAttribute('name')
+                        .toLowerCase()
+                        .split('-')[1];
+                selectElement.appendChild(dropDownOption);
+                options.forEach(function (element) {
+                    if (kommunicateCommons.isObject(element)) {
+                        dropDownOption = document.createElement('option');
+                        dropDownOption.setAttribute('value',element.value);
+                        dropDownOption.textContent =
+                            element.value.charAt(0).toUpperCase() +
+                            element.value.slice(1);
+                        selectElement.appendChild(dropDownOption);
+                    } else {
+                        throw new TypeError(
```

Comment:
what will the chat widget stop working if this error gets thrown?

Reply:
ratified